jax/experimental/jax2tf/tests/jax2tf_test.py

```diff
@@ -516,6 +517,21 @@ def jax_fn_array(x):
         tf_fn_array(np.array([3, 4, 5])), np.array([4.5, 10, 17.5],
                                                    jnp.bfloat16))
 
+  def test_enable_xla(self):
+    # Tests that enable_xla flag is properly scoped to a conversion.
+    def fun(x):
+      # Can be converted only if enable_xla is on, due to negative padding.
+      return lax.pad(x, np.float32(0), [(-1, 0, 0), (0, 0, 0)])
+
+    tf_fun_with_xla = jax2tf.convert(fun, enable_xla=True)
+    tf_fun_without_xla = jax2tf.convert(fun, enable_xla=False)
+    x = np.ones((2, 3), dtype=np.float32)
+
+    self.assertAllClose(fun(x), tf_fun_with_xla(x))
+    with self.assertRaisesRegex(NotImplementedError,
+                                "Call to pad can only be converted through TFXLA"):
+      tf_fun_without_xla(x)
```

Comment:
For completeness / pedantry, consider converting a different function in the opposite order (for values of `_enable_xla`).

Reply:
Done.